src/py/flwr/common/record/parametersrecord.py

```diff
@@ -117,3 +117,17 @@ def __init__(
                 self[k] = array_dict[k]
                 if not keep_input:
                     del array_dict[k]
+
+    def count_bytes(self) -> int:
+        """Return number of Bytes stored in this object."""
+        num_bytes = 0
```

Comment:
I think we can just count the bytes by counting the bytes of each array and key, right?